webpack.playground.config.js

```diff
@@ -11,14 +11,16 @@ const WIDGET_RC = '.widgetrc';
 
 let widgetRc = {
   widgetOptions: {
-    baseUrl: 'https://{yourOktaDomain}',
+    baseUrl: 'http://localhost:3000',
     logo: '/img/logo_widgico.png',
     logoText: 'Windico',
     features: {
       router: true,
       rememberMe: true,
+      securityImage: true,
```

Comment:
security image has not worked for me before locally... not sure if that is fixed or maybe it was just me, but if it doesn't work easily for other people, should we then leave it disabled?

Reply:
why do we need this to be true ?  securityImage does not work for new pipeline anyways

Reply:
- @mauriciocastillosilva-okta it does given we mocked `http://localhost:3000/login/getimage` request
- @nikhilvenkatraman-okta the `widgetrc` is not dedicated for OIE though. but I'll remove it from the default generator now.